app/web/features/communities/members/CommunityMembersList.tsx

```diff
@@ -46,50 +22,73 @@ const LoadMoreButtonWrapper = styled("div")(({ theme }) => ({
 
 export default function CommunityMembersList({
   communityId,
+  memberCount,
 }: {
   communityId: Community.AsObject["communityId"];
+  memberCount?: Community.AsObject["memberCount"];
 }) {
   const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+  const PAGE_SIZE = 20;
 
-  const { data, isLoading, isFetching, error, fetchNextPage, hasNextPage } =
-    useListMembers(communityId);
+  const [pageNumber, setPageNumber] = useState(1);
 
-  const memberUserIdsList = data?.pages.flatMap(
-    (page) => page.memberUserIdsList,
-  );
+  const { data, isFetching, isLoading, error, fetchNextPage } = useListMembers({
+    communityId,
+    pageSize: PAGE_SIZE,
+  });
+
+  const memberUserIdsList = data?.pages
+    ? data.pages[pageNumber - 1]?.memberUserIdsList
+    : [];
+  const numPages = Math.ceil((memberCount ?? 0) / PAGE_SIZE);
+
+  const handlePageNumberChange = (
+    event: React.ChangeEvent<unknown>,
+    value: number,
+  ) => {
+    if (value > pageNumber) {
+      fetchNextPage();
+    }
```

Comment:
OK, I don't understand the page numbering (how you are computing where to jump if I click on a given page number).

I think it's better if we have infinite pages for the time being, given we can't compute the exact page numbers.

On prod we'll have a bunch of gaps due to deleted/banned users, so we can't reliably compute the right number. On next it has "1, 2, 3, ... 9" in global, but the last page never loads (I know you changed the order, but the API didn't change, it still does the same thing, just opposite order, I think?)